unicorn_mode/build_unicorn_support.sh

```diff
@@ -182,7 +182,8 @@ echo "[+] Configuration complete."
 echo "[*] Attempting to build unicornafl (fingers crossed!)..."
 
 $MAKECMD clean  # make doesn't seem to work for unicorn
-UNICORN_QEMU_FLAGS="--python=$PYTHONBIN" $MAKECMD -j$CORES || exit 1
+#UNICORN_QEMU_FLAGS="--python=$PYTHONBIN" $MAKECMD -j$CORES || exit 1
+UNICORN_QEMU_FLAGS="--python=$PYTHONBIN" $MAKECMD || exit 1
```

Comment:
The unicorn-engine Makefile will automatically set `-j4` if not specified otherwise. Maybe we should set `-j1` here, at least until the race condition is fixed?
Probably open a new issue for it, too.